website/static/js/citationList.js

```diff
@@ -14,8 +14,19 @@ var STYLES = {
 
 var ctx = window.contextVars;
 
+var escapeHTML = function(s) {
```

Comment:
Specifically, `$osf.htmlEscape` (unless there are specific advantages to your alternative that you think should be considered)